### PR TITLE
Added a minimap that displays the whole level

### PR DESCRIPTION
**Pull request by hwoarangmy**
Note that to use it, you need to go to the "settings" menu in the "game" tab

### CODE REVIEW

**Comment by eugeneloza** [comment]
I saw no difference between minimaps modes? Or did I do something wrong?

**Comment by akien-mga** [comment]
@eugeneloza If you're playing 0.7.0, it's not in there yet as it wasn't merged. I think it's available in @hwoarangmy's last WIP though.

**Comment by eugeneloza** [comment]
@akien-mga I've used latest snapshot from freegamedev. And the option to select different minimaps is present in the settings.

**Comment by hwoarangmy** [comment]
You have to choose "MiniMapDrawnFull" in the settings menu, game tab.

File: source/entities/Tile.cpp [hunk 1]
```diff
@@ -1620,7 +1620,7 @@ bool Tile::addTreasuryObject(TreasuryObject* obj)
     if(!getIsOnServerMap())
     {
         // On client side, we add the entity to tile. Merging is relevant on server side only
-        mEntitiesInTile.push_back(obj);
+        addEntity(obj);
```
Comment (akien-mga) [inline source/entities/Tile.cpp:1640]: Shorter and clearer is always better :)

**Comment by akien-mga** [comment]
Builds and runs fine, sorry for the excessive delay. I still have to review the commit adding the new minimap for my personal culture, but I trust that it's good code :)

@hwoarangmy As long as @Bertram25 is MIA, if I take too long to react on a PR, don't hesitate to merge it yourself so that it does not hinder you. The PRs can still be reviewed post-merge, and having them is useful to get notified of what's up in the repo :)

**Comment by hwoarangmy** [comment]
ok. I will keep that in mind ^^

**Pull request merged**